typescript/src/string/lcfirst.ts

```diff
@@ -0,0 +1 @@
+export const lcfirst = (text: string): string => text.charAt(0).toLowerCase() + text.slice(1);
```

Comment:
tambahin docstring, refer ke dokumentasi PHP buat susunan kalimatnya

Reply:
done in [295b088](https://github.com/teknologi-umum/pehape/pull/65/commits/295b08812b93349f40d53bd6a9b7d16c3d644548)